notebooks/hello-universe/Regressing_3D-HST_galaxy_redshift_with_decision_trees.ipynb

```diff
@@ -1,1446 +0,0 @@
-      "text/plain": [
-       "<Figure size 288x288 with 1 Axes>"
-      ]
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
-    }
-   ],
-   "source": [
-    "fig, ax = plt.subplots(1, 1, figsize=(4, 4))\n",
-    "\n",
-    "ax.scatter(df.z_spec, df.lmass, alpha=0.2, color='grey')\n",
-    "ax.set_xlim(0, 2)\n",
-    "ax.set_ylim(7, 12)\n",
-    "ax.set_xlabel(r'$z_{\\rm spec}$')\n",
-    "ax.set_ylabel(r'$\\log{(M_{*})}\\,\\,[M_{\\odot}]$')\n",
-    "\n",
-    "plt.show()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "4c274dd5",
-   "metadata": {},
-   "source": [
-    "By inspection of this plot, we will apply a cut of log(M)>9 Solar masses to the dataset to remove sources which do not have complete coverage across redshift."
```

Comment:
consider rephrasing:  "apply a cut of log(M)>9" -> "only keep those with log(M)>9"